package/MDAnalysis/coordinates/PDB.py

```diff
@@ -1179,10 +1182,36 @@ def _write_timestep(self, ts, multiframe=False):
             vals['tempFactor'] = temp if temp is not None else 0.0
             vals['segID'] = segid[:4]
             vals['element'] = guess_atom_element(atom.name.strip())[:2]
-
             # .. _ATOM: http://www.wwpdb.org/documentation/format32/sect9.html
+            
+            if i != 0 and vals['chainID'] != prev_atom['chainID'] \
+                and vals['chainID'] != " " and prev_atom['chainID'] != " ":
+                end_atom = {}
+                end_atom['serial'] = int(str(prev_atom['serial'] + 1)[-5:])
+                 # check for overflow here?
+                end_atom['resName'] = prev_atom['resName']
+                end_atom['chainID'] = prev_atom['chainID']
+                end_atom['resSeq'] = prev_atom['resSeq']
+                end_atom['iCode'] = prev_atom['iCode']
+                self.TER(end_atom)
+                serial_count += 1
+                vals['serial'] = int(str(serial_count + 1)[-5:])
+                # check for overflow here?
+          
             self.pdbfile.write(self.fmt['ATOM'].format(**vals))
+            prev_atom = copy.deepcopy(vals)
+            serial_count += 1
+
         if multiframe:
+            ter_atom = {}
```

Comment:
The TER at the end of a frame should be written even if `multiframe` is `False`.